test/functional/test_framework/messages.py

```diff
@@ -428,7 +443,10 @@ def __repr__(self):
                   bytes_to_hex_str(self.scriptPubKey))
 
 
-class CTransaction():
+class CTransaction:
+    __slots__ = ("hash", "nLockTime", "nVersion", "sha256", "vin", "vout",
+                 "wit", "nType", "vExtraPayload")
```

Comment:
14305:
```suggestion
                 "nType", "vExtraPayload")
```

Reply:
The above mentioned changes applied, Thank you